docs/comparison.md

```diff
@@ -166,20 +284,77 @@ MycroftAI also provides some images, we do not recommend using any of these but
-| TTS - Cloud                                                                                                |                         default: **Mimic 2**            |                       default: **Mimic 2**                        | default: **Mimic 2** |
-| **Smart Speaker - Functionality**                                                                          |
-| Music player connectivity<br>*use of external applications<br>on other devices*                            |                     No                   |           Yes<br>*MPD<br>Local Files*           | No<br>*manual installation?* |
-| Music player sync                                                                                          |                                                     No             |                       No                        | No |
-| HomeAssistant integration                                                                                  |                                                            ?   |                     Yes<br>Mycroft Skill                     | ? |
-| Camera support                                                                                             |                                                                          No                      |                     ?                     | ? |
\ No newline at end of file
+
+
+## Hardware
+
+### OpenVoiceOS
+
+|                              | ovos-buildroot                | ovos-arch            | ovos-raspbian                        |
+|------------------------------|-------------------------------|----------------------|--------------------------------------|
+| **Hardware - Compatibility** |
+| Raspberry Pi 3B              | **WIP**                           | No                   | Yes                                  |   
+| Raspberry Pi 3B+             | **WIP**                           | No                   | Yes                                  | 
+| Raspberry Pi 3A+             | **WIP**                           | No                   | Yes                                  |   
+| Raspberry Pi 4               | Yes                           | Yes                  | Yes                                  | 
+| X86_64                       | *planned*                     | No                   | No                                   |                     
+| Mark-1                       | **WIP**                       | No                   | No<br>*possibly in future*           |                        
+| Mark-2                       | **WIP**<br>(no leds)          | **WIP**<br>(no leds) | partial<br>(no gui + manual install) |           
+| Mark-2  (dev kit)            | Yes                           | Yes                  | partial<br>(no gui + manual install) |           
+| **Hardware - Peripherals**   |
```

Comment:
These I would also like to support out of the box.  I am looking at how the buildroot image detects them and will try and add that to the code.  Maybe "WIP"

Reply:
i'd like to only use WIP if some functionality is there already, eg, HA plugin but no Voice interface yet

for the other i'd prefer to use "planned" if work didn't start yet

Reply:
That works for me